test/e2e/framework/framework.go

```diff
@@ -41,6 +41,7 @@ func beforeSuite() {
 	}
 
 	queryAndUpdateGlobalnetStatus()
+	framework.TestContext.OperationTimeout = 200
```

Comment:
This would override a setting on the command line. Why not make this the default in shipyard?

Reply:
Sure @tpantelis. I pushed the following PR in shipyard - https://github.com/submariner-io/shipyard/pull/250